mne/io/eeglab/eeglab.py

```diff
@@ -6,11 +6,14 @@
 import os.path as op
 
 import numpy as np
+import h5py  # Added to read newer Matlab files (7.3 and later)
```

Comment:
you cannot add a strong dependency like this.

You need to use a nested import.

also you'll need to update the continuous integration tools (travis, appveyor)